modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -17,16 +18,66 @@ const defaultExpectedEvents = [
 ];
 
 describe('browser.tinymce.core.content.EditorContentTest', () => {
+  const toHtml = (node: AstNode): string => {
+    const htmlSerializer = HtmlSerializer({});
+    return htmlSerializer.serialize(node);
+  };
```

Comment:
Do we need the extra variable for something. I think this is shorter and readable? I know that the original function had it as well. So up to you what you prefer.

```suggestion
  const toHtml = (node: AstNode): string => HtmlSerializer({}).serialize(node);
```

Reply:
Thanks for catching that, definitely prefer the shortened version.